### PR TITLE
MTM-52999: announce authentication with access tokens from authorization

### PR DESCRIPTION
**Pull request by chbakar**
servers

### CODE REVIEW

File: content/release-10-18-0/whatsnew-10-18-0.md [hunk 1]
```diff
@@ -7,3 +7,11 @@ layout: bundle
 Release 10.18.0 includes the following new features or major feature enhancements.
 
 ### Application enablement
+
+
+### Authentication
+
+#### Authentication with OAuth2 access tokens from authorization servers
+
+Access tokens from third party authorization servers can be used to authenticate in Cumulocity IoT. The SSO configuration is extended to allow the configuration of such authentication. Configuring this authentication,
```
Comment (BeateRixen) [inline content/release-10-18-0/whatsnew-10-18-0.md:16]:
```suggestion
It is now possible to use access tokens from third-party authorization servers to authenticate in Cumulocity IoT. The SSO configuration has been extended accordingly. Configuring this authentication type,
```

Comment (BeateRixen) [reply content/release-10-18-0/whatsnew-10-18-0.md:16]: > 
On top of standard SSO, Cumulocity IoT now also allows you to access the platform resources using access tokens from third-party authorization server directly as a Bearer token. The SSO configuration has been extended accordingly. For details refer to [Configuring authentication with OAuth2 access tokens from authorization servers](/users-guide/administration/#configuring-authentication-with-oauth2-access-tokens-from-authorization-servers).

File: content/release-10-18-0/whatsnew-10-18-0.md [hunk 2]
```diff
@@ -7,3 +7,11 @@ layout: bundle
 Release 10.18.0 includes the following new features or major feature enhancements.
 
 ### Application enablement
+
+
+### Authentication
+
+#### Authentication with OAuth2 access tokens from authorization servers
+
+It is now possible to use OAuth2 access tokens from third-party authorization servers to access Cumulocity REST API. The SSO configuration has been extended accordingly. Configuring this authentication type,
```
Comment (BeateRixen) [inline content/release-10-18-0/whatsnew-10-18-0.md:16]:
```suggestion
On top of standard SSO, Cumulocity IoT now also allows you to access the platform resources using access tokens from third-party authorization server directly as a Bearer token. The SSO configuration has been extended accordingly. For details refer to [Configuring authentication with OAuth2 access tokens from authorization servers](https://github.com/users-guide/administration/#configuring-authentication-with-oauth2-access-tokens-from-authorization-servers).
```

File: content/release-10-18-0/whatsnew-10-18-0.md [hunk 3]
```diff
@@ -7,3 +7,10 @@ layout: bundle
 Release 10.18.0 includes the following new features or major feature enhancements.
 
 ### Application enablement
+
+
+### Authentication
+
+#### Authentication with OAuth2 access tokens from authorization servers
+
+On top of standard SSO, Cumulocity IoT now also allows you to access the platform resources using access tokens from third-party authorization server directly as a Bearer token. The SSO configuration has been extended accordingly. For details refer to [Configuring authentication with OAuth2 access tokens from authorization servers](https://github.com/users-guide/administration/#configuring-authentication-with-oauth2-access-tokens-from-authorization-servers).
```
Comment (BeateRixen) [inline content/release-10-18-0/whatsnew-10-18-0.md:16]:
```suggestion
On top of standard SSO, Cumulocity IoT now also allows you to access the platform resources using access tokens from third-party authorization server directly as a Bearer token. The SSO configuration has been extended accordingly. For details refer to [Configuring authentication with OAuth2 access tokens from authorization servers](https://cumulocity.com/guides/10.18.0/users-guide/administration/#configuring-authentication-with-oauth2-access-tokens-from-authorization-servers).
```

File: content/release-10-18-0/whatsnew-10-18-0.md [hunk 4]
```diff
@@ -7,3 +7,10 @@ layout: bundle
 Release 10.18.0 includes the following new features or major feature enhancements.
 
 ### Application enablement
+
+
+### Authentication
+
+#### Authentication with OAuth2 access tokens from authorization servers
+
+On top of standard SSO, Cumulocity IoT now also allows you to access the platform resources using access tokens from third-party authorization server directly as a Bearer token. The SSO configuration has been extended accordingly. For details refer to [Configuring authentication with OAuth2 access tokens from authorization servers](https://cumulocity.com/guides/10.18.0/users-guide/administration/#configuring-authentication-with-oauth2-access-tokens-from-authorization-servers).
```
Comment (BeateRixen) [inline content/release-10-18-0/whatsnew-10-18-0.md:16]:
```suggestion
On top of standard SSO, Cumulocity IoT now also allows you to access the platform resources using access tokens from third-party authorization server directly as a Bearer token. The SSO configuration has been extended accordingly. For details refer to [Configuring single sign-on > Configuring authentication with OAuth2 access tokens from authorization servers](https://cumulocity.com/guides/10.18.0/users-guide/administration/#configuring-authentication-with-oauth2-access-tokens-from-authorization-servers) in the *User guide*.
```

**Review by BeateRixen — APPROVED** [review]

**Review by Michal-Bak — APPROVED** [review]

**Pull request merged**